ruby-challenges.rb

```diff
@@ -22,6 +37,13 @@
 # Expected output: 'bby Rd'
 
 
+def no_vowels (str) 
+   
+   str[/[aeiou](.)\1/] do
```

Comment:
This seems to have a different output than the expected output.  Ruby has a really great delete built-in method that will remove the characters you place inside of it.  Would suggest looking into it.